README.md

```diff
@@ -0,0 +1,69 @@
+# SauceDemo Automated Tests
```

Comment:
In order to run tests on CI, please copy config file from this https://github.com/ovartem/playwright_tests_with_po/blob/master/.github/workflows/playwright.yml. Except, please change line 15 to `image: mcr.microsoft.com/playwright:v1.48.1-jammy`